package.json

```diff
@@ -177,8 +177,5 @@
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
       "pre-push": "if git-branch-is main; then yarn pre-push; fi"
     }
-  },
-  "engines": {
-    "node": "12.x"
```

Comment:
Because Codesandbox uses node v10

Reply:
I could give it a range I suppose. Either `"node" : ">=10.x <=12.x"` or `"node" : ">=10.x"`

Reply:
The range is probably the safest path, right?